experimental/sql/dialects/ibis_dialect.py

```diff
@@ -318,17 +291,18 @@ class Aggregation(Operation):
   table = SingleBlockRegionDef()
```

Comment:
Please update the docu. Make sure you discuss the semantics of what happens if `by` and `metrics` is empty, respectively, vs when it is not.